operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md

```diff
@@ -160,51 +165,56 @@ Run these commands on the rebuilt node.
     * This command assumes you have set the variables from [the prerequisites section](../Rebuild_NCNs.md#Prerequisites).
 
         ```bash
-        ncn# dig +short ${NODE}.can
-        10.103.8.11
+        ncn# dig +short ${NODE}.cmn
+        10.101.5.27
         ```
 
 2. Confirm the output from the dig command matches the interface.
 
    * If the IP addresses match, proceed to the next step. If they do not match, continue with the following sub-steps.
 
         ```bash
-        ip addr show vlan007
+        ip addr show bond0.cmn0
         ```
 
         Example Output:
 
         ```screen
-        15: vlan007@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
-            link/ether b8:59:9f:2b:2f:9e brd ff:ff:ff:ff:ff:ff
-            inet 10.103.8.11/24 brd 10.103.8.255 scope global vlan007
-               valid_lft forever preferred_lft forever
-            inet6 fe80::ba59:9fff:fe2b:2f9e/64 scope link
-               valid_lft forever preferred_lft forever
+        9: bond0.cmn0@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9000 qdisc noqueue state UP group default qlen 1000
+           link/ether a4:bf:01:65:6a:aa brd ff:ff:ff:ff:ff:ff
+           inet 10.101.5.27/25 brd 10.101.5.127 scope global bond0.cmn0
+              valid_lft forever preferred_lft forever
+           inet6 fe80::a6bf:1ff:fe65:6aaa/64 scope link
+              valid_lft forever preferred_lft forever
         ```
 
-        1. Change the IP address for `vlan007` if necessary.
+        1. Change the IP address and netmask for `bond0.cmn0` if necessary.
+
+            ```bash
+            ncn# cray sls networks list --format json | jq -r '.[] | select(.Name == "CMN") | .ExtraProperties.CIDR'
+            10.101.5.0/25
```

Comment:
This is not going to be the IP address of bond0.cmn0.  This is the network CIDR.  So it will give you the netmask that you need.  But The way this is right now, it looks to me like we are saying to set the IP address for bond0.cmn0 to `10.101.5.0/25`

Reply:
We discussed further - these sections should be removed from 1.2 given that we are no longer DHCP'ing on those interfaces which in the past caused these IP to get allocated from the pool rather than the reserved IPs. Now we are getting the IPs from the BSS metadata and not using DHCP.